errors/manifest.json

```diff
@@ -549,12 +549,12 @@
           "path": "/errors/sharp-version-avif.md"
         },
         {
-          "title": "script-in-document-page",
-          "path": "/errors/no-script-in-document-page.md"
+          "title": "no-script-in-document",
+          "path": "/errors/no-script-in-document.md"
```

Comment:
Note we'll probably want redirects here 👍

Reply:
I had wondered about redirects here, but I didn't see one added for the [previous PR changing URLs](https://github.com/vercel/next.js/pull/33968), but I can totally make that happen (for the previous PR too).

Reply:
Yes!